functions/package.json

```diff
@@ -5,7 +5,7 @@
     "lint": "prettier --write \"./**/*.{js,jsx,ts,tsx,json,css,scss,md}\""
   },
   "engines": {
-    "node": "10"
+    "node": "14"
```

Comment:
That's to gain an ability to use ?. optional chaining. Also this is recommended by google: [docs](https://cloud.google.com/functions/docs/concepts/nodejs-runtime).
When I finish #2351 we can change this back to 10 (if any problems arise) as TS supports optional chaining and much more.